doc/tutorials/starship.rst

```diff
@@ -0,0 +1,99 @@
+.. _starship:
+
+Running Jobs with StarshipEngine
+################################
+
+.. sectionauthor:: Zeid Zabaneh <zeid@xanadu.ai>
+
+In this section, we provide a tutorial of the **StarshipEngine**, an engine used to connect to the Strawberry Fields platform and execute jobs remotely (e.g. on a quantum chip).
+
+Configuring StarshipEngine
+==========================
+
+Before using StarshipEngine, you need to configure the hostname and authentication token that will provide you access to the API. The easiest way is to create a configuration file named ``config.toml`` in your working directory. A typical file looks like this:
+
+.. code-block:: console
+
+   [api]
+   hostname = "platform.example.com"
+   authentication_token = "ElUFm3O6m6q1DXPmpi5g4hWEhYHXFxBc"
+
+You can generate this file interactively by using the ``starship`` command as follows, answering the questions in the prompts.
+
+.. code-block:: console
+
+   (strawberryfields)$ starship --reconfigure
+   Please enter the hostname of the server to connect to: [platform.example.com]
+   Please enter the authentication token to use when connecting: [] ElUFm3O6m6q1DXPmpi5g4hWEhYHXFxBc
+   Would you like to save these settings to a local cofiguration file in the current directory? [Y/n] y
+   Writing configuration file to current working directory...
+
+
+To test connectivity, you can use the following command:
+
+.. code-block:: console
+
+   (strawberryfields)$ starship --hello
```

Comment:
Nice, agreed this would be useful to have. Would make tests easier too

Reply:
I like the `--hello` flag :laughing: